src/main/java/core/basesyntax/strategy/OperationHandler.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationHandler {
+    public void handle (Storage storage, FruitTransaction transaction);
```

Comment:
There should not be a space between the method name `handle` and the opening parenthesis.